.stylelintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "plugins": [
+    "stylelint-order"
+  ],
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-styled-components",
+    "stylelint-config-rational-order",
+    "stylelint-config-prettier"
+  ],
+  "defaultSeverity": "warning",
+  "rules": {
+    "order/properties-order": [
+      [],
+      {
+        "severity": "warning"
+      }
+    ],
+    "plugin/rational-order": [
+      true,
+      {
+        "severity": "warning"
+      }
+    ],
+    "value-keyword-case": null,
+    "function-name-case": null,
+    "custom-property-empty-line-before": null,
+    "unit-no-unknown": null,
+    "rule-empty-line-before": ["always", {
+      "ignore": "first-nested"
+    }]
+  }
+}
+
```

Comment:
Maybe you could add filename extension `.stylelintrc.js`, or `.stylelintrc.json` to get syntax checking and highlighting

Reply:
that's a good point, will change that